tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -1,10 +1,10 @@
-#ifndef USER_DATA_SIZE
-#define USER_DATA_SIZE 1000
-#endif
+/****************************************************************/
```

Comment:
Please could you not include these changes in future PRs, it's difficult for me to see whether you've actually changed the content of the comment

Reply:
I don't understand.  You appear to be seeing two adjacent changes made in two different commits in the PR that are being displayed together.